dbms/src/Storages/StorageDisaggregatedRemote.cpp

```diff
@@ -515,13 +515,6 @@ std::variant<DM::Remote::RNWorkersPtr, DM::SegmentReadTaskPoolPtr> StorageDisagg
     const auto & executor_id = table_scan.getTableScanExecutorID();
 
     auto rs_operator = buildRSOperator(db_context, column_defines);
-    {
-        DM::ANNQueryInfoPtr ann_query_info = nullptr;
-        if (table_scan.getANNQueryInfo().query_type() != tipb::ANNQueryType::InvalidQueryType)
-            ann_query_info = std::make_shared<tipb::ANNQueryInfo>(table_scan.getANNQueryInfo());
-        if (ann_query_info != nullptr)
-            rs_operator = wrapWithANNQueryInfo(rs_operator, ann_query_info);
-    }
```

Comment:
Remove because the logic is duplicated within `RSOperator::build` https://github.com/pingcap/tiflash/pull/9503